### PR TITLE
Feature/Option to use OR queries on security-data/alerts search

### PR DESCRIPTION
**Pull request by timabrmsn**
Adds `--or-query` option to make the passed in filters be processed in an OR group for both alert and file event searches.

### CODE REVIEW

File: src/code42cli/cmds/securitydata.py [hunk 1]
```diff
@@ -163,9 +163,20 @@ def clear_checkpoint(state, checkpoint_name):
 @security_data.command()
 @file_event_options
 @search_options
+@click.option(
+    "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
```
Comment (alanag13) [inline src/code42cli/cmds/securitydata.py:167]: I do think there is potential usefulness to not having this restriction, but it probably warrants further discussion and should just stay this way for now. We can discuss further once we pursue "merging" cli options with saved search/advanced search.

**Comment by alanag13** [comment]
Needs tests and testing instructions, but otherwise LGTM.

File: src/code42cli/cmds/alerts.py [hunk 2]
```diff
@@ -155,15 +155,22 @@ def clear_checkpoint(state, checkpoint_name):
...
+    cli_state, format, begin, end, advanced_query, use_checkpoint, or_query, **kwargs
+):
     """Search for alerts."""
     output_logger = logger_factory.get_logger_for_stdout(format)
     cursor = _get_alert_cursor_store(cli_state.profile.name) if use_checkpoint else None
     handlers = ext.create_handlers(
         cli_state.sdk, AlertExtractor, output_logger, cursor, use_checkpoint
     )
     extractor = _get_alert_extractor(cli_state.sdk, handlers)
+    if or_query:
```
Comment (unparalleled-js) [inline src/code42cli/cmds/alerts.py:172]: any reason not to just do `extractor.use_or_query = or_query` ?

Comment (timabrmsn) [reply src/code42cli/cmds/alerts.py]: nope, I think i just copied the logic from security-data and then removed the extra line.

File: src/code42cli/cmds/securitydata.py [hunk 3]
```diff
@@ -176,6 +187,9 @@ def search(
         state.sdk, FileEventExtractor, output_logger, cursor, use_checkpoint
     )
     extractor = _get_file_event_extractor(state.sdk, handlers)
+    if or_query:
```
Comment (alanag13) [inline src/code42cli/cmds/securitydata.py:190]:
```python
extractor.use_or_query = or_query
```

**Pull request merged**